pytorch_lightning/core/grads.py

```diff
@@ -3,28 +3,25 @@
 """
 from typing import Dict
 
-from torch import nn
+import torch
 
 
-class GradInformation(nn.Module):
+class GradInformation(torch.nn.Module):
 
     def grad_norm(self, norm_type: float) -> Dict[str, int]:
```

Comment:
Would you be so kind to also add a brief docstring describing which values are allowed for "norm type" (p >= 1) and will be returned in the dict?

Reply:
... and also add a changelog entry for this bugfix?

Reply:
The annotation should also be changd to Dict[str, float]